components/Footer/SpotifyPlayer/SpotifyPlayer.tsx

```diff
@@ -137,6 +146,23 @@ export const SpotifyPlayer: React.FC<SpotifyPlayerProps> = (props) => {
     }
   }, []);
 
+  useEffect(() => {
+    if (!trackState) {
+      return;
+    }
+
+    const interval = setInterval(
+      () =>
+        !isPaused &&
+        setCurrentTrackPositionUsecase(
+          trackState.position + +new Date() - trackState?.timestamp
+        ),
+      1000
+    );
```

Comment:
Одна строчка это классно, но она совсем не читабельно, не понятно что к чему. Я бы писал как-то так:
```suggestion
     () => {
        if (isPaused) {
          return;
        }

        const { timestamp, position: lastStateChangePosition } = trackState;
        const diffAfterLastStateChange = new Date().valueOf() - timestamp;
        const newTrackPosition = lastStateChangePosition + diffAfterLastStateChange;

        return setCurrentTrackPositionUsecase(newTrackPosition);
      },  
      1000
    );
```